src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  res.send(userService.getAllUsers());
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = userService.createUser(name);
+
+  res.statusCode = 201;
+  res.send(user);
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+
+  if (!userService.getUserById(id)) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  userService.deleteUser(id);
+
+  res.sendStatus(204);
+};
```

Comment:
In the 'remove' function, you're checking if the user exists before deleting it, which is good. However, you're not handling the case where the 'deleteUser' function might fail. Consider adding error handling for this case.